tensorflow/core/common_runtime/gpu/gpu_util.cc

```diff
@@ -378,10 +395,12 @@ void GPUUtil::CopyCPUTensorToGPU(const Tensor* cpu_tensor,
     }
   }
 
+  bool merge_HtoD = gpu_device->merge_HtoD_copy_stream();
```

Comment:
```suggestion
 const bool merge_host_to_device_stream = gpu_device->merge_HtoD_copy_stream();
```

Reply:
Also, can you move this to before Line 342, and use it afterwards in this function.

Similar for the other two copy functions